voice/voice-ai/x/Performance/TimeLogger.swift

```diff
@@ -182,21 +190,23 @@ class TimeLogger {
         appResFirstTimestamp = now()
     }
 
-    func setAppResEnd() {
+    func setAppResEnd(test: Bool = false) {
         appResEndTimestamp = now()
-        if appResFirstTimestamp == 0 {
+        if appResFirstTimestamp == 0, !test {
             appResFirstTimestamp = appResEndTimestamp
         }
     }
     
-    func setInferenceStats(requestNumMessages: Int32 = 0, requestNumUserMessages: Int32 = 0, requestTokens: Int32 = 0, responseTokens: Int32 = 0, requestMessage: String = "", responseMessage: String = "", cancelled: Bool = false, completed: Bool = true, error: String = "") {
+    func setInferenceStats(requestNumMessages: Int32 = 0, requestNumUserMessages: Int32 = 0, requestTokens: Int32 = 0, responseTokens: Int32 = 0, requestMessage: String = "", responseMessage: String = "", cancelled: Bool = false, completed: Bool = true, error: String = "", test: Bool = false) {
```

Comment:
using one or two class member variables (e.g. test mode), you can define a set of new behaviors and implement in functions